sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/api/Azure.Monitor.OpenTelemetry.LiveMetrics.netstandard2.0.cs

```diff
@@ -1,3 +1,22 @@
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics
+{
+    public partial class LiveMetricsExporterOptions : Azure.Core.ClientOptions
+    {
+        public LiveMetricsExporterOptions() { }
+        public string ConnectionString { get { throw null; } set { } }
+        public Azure.Core.TokenCredential Credential { get { throw null; } set { } }
```

Comment:
Does live metrics support AAD? If not remove changes related to it.

Reply:
Yes it does.